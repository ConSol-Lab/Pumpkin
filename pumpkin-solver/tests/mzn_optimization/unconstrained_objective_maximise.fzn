var 1..5: objective :: output_var;

solve maximize objective;
