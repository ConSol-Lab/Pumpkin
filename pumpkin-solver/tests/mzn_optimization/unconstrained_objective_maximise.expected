objective = 3;
----------
objective = 4;
----------
objective = 5;
----------
==========
